src/log/access_log.cc

```diff
@@ -503,6 +512,14 @@ fvdbDumpTable(StoreEntry * e, hash_table * hash)
     }
 }
 
+static void
+fvdbDumpTable(StoreEntry * e, const fvdb_table_t& hash)
+{
+    for (const auto & i : hash)
+        storeAppendPrintf(e, "%9d %.*s\n", i.second,
+            i.first.length(), i.first.rawContent());
```

Comment:
FWIW, I only see `%9` formatting in fvdbDumpTable(). That formatting is trivial to accomplish with `std::setw(9)`. Thus, I see no reason to avoid the current default interface for writing formatted output to StoreEntry -- PackableStream (and several reasons to use that default, of course). Please correct me if I missed something.

The presence of that alignment-based formatting is itself unwelcomed, but fixing cache manager output format is outside this specific PR scope IMO.

Reply:
Merged @yadij 's suggestion